src/liquid/components/ld-bg-cells/readme.md

```diff
@@ -0,0 +1,282 @@
+  <div class="ld-bg-cells__content">
+    <p class="ld-paragraph">type: millipore</p>
+  </div>
+</div>
+<div class="ld-bg-cells ld-bg-cells--qa-x2f-qc box2-16/9">
+  <div class="ld-bg-cells__pattern"></div>
+  <div class="ld-bg-cells__content">
+    <p class="ld-paragraph">type: qa-x2f-qc</p>
+  </div>
+</div>
+<div class="ld-bg-cells ld-bg-cells--safc box2-16/9">
+  <div class="ld-bg-cells__pattern"></div>
+  <div class="ld-bg-cells__content">
+    <p class="ld-paragraph">type: safc</p>
+  </div>
+</div>
+<div class="ld-bg-cells ld-bg-cells--sigma-aldrich box2-16/9">
+  <div class="ld-bg-cells__pattern"></div>
+  <div class="ld-bg-cells__content">
+    <p class="ld-paragraph">type: sigma-aldrich</p>
+  </div>
+</div>
+<div class="ld-bg-cells ld-bg-cells--t box2-16/9">
+  <div class="ld-bg-cells__pattern"></div>
+  <div class="ld-bg-cells__content">
+    <p class="ld-paragraph">type: t</p>
+  </div>
+</div>
+<div class="ld-bg-cells ld-bg-cells--tile box2-16/9">
+  <div class="ld-bg-cells__pattern"></div>
+  <div class="ld-bg-cells__content">
+    <p class="ld-paragraph">type: tile</p>
+  </div>
+</div>
+
+<style>
+  .box2-16\/9 {
+    height: 12.3046875rem;
+    width: 21.875rem;
+  }
+
+  .box2-16\/9 .ld-bg-cells__content {
+    align-items: center;
+    color: var(--ld-col-wht);
+    display: flex;
+    justify-content: center;
+  }
+</style> -->
+
+## Custom position, size and colors
+
+{% example %}
+<ld-bg-cells class="box3-16/9"></ld-bg-cells>
+
+<style>
+  .box3-16\/9 {
+    --ld-bg-cells-background-color: var(--ld-col-vm2);
+    --ld-bg-cells-foreground-color: var(--ld-col-vm4);
+    --ld-bg-cells-position: bottom left;
+    --ld-bg-cells-size: 150%;
+
+    margin: 0 0 calc(var(--ld-sp-24) * -1 + 1px) !important;
+    padding-top: 56.25%;
+  }
+</style>
+{% endexample %}
+
+<!-- CSS component -->
+
+<!-- <div class="ld-bg-cells ld-bg-cells--safc box3-16/9">
+  <div class="ld-bg-cells__pattern"></div>
+</div>
+
+<style>
+  .box3-16\/9 {
+    --ld-bg-cells-background-color: var(--ld-col-vm2);
+    --ld-bg-cells-foreground-color: var(--ld-col-vm4);
+    --ld-bg-cells-position: bottom left;
+    --ld-bg-cells-size: 150%;
+
+    margin: 0 0 calc(var(--ld-sp-24) * -1 + 1px) !important;
+    padding-top: 56.25%;
+  }
+</style> -->
+
+## Repeating cells pattern
+
+{% example %}
+<ld-bg-cells class="box4-16/9" repeat></ld-bg-cells>
+
+<style>
+  .box4-16\/9 {
+    margin: 0 0 calc(var(--ld-sp-24) * -1 + 1px) !important;
+    padding-top: 56.25%;
+  }
+</style>
+{% endexample %}
+
+<!-- CSS component -->
+
+<!-- <div class="ld-bg-cells ld-bg-cells--safc box4-16/9">
+  <div class="ld-bg-cells__pattern ld-bg-cells__pattern--repeat"></div>
+</div>
+
+<style>
+  .box4-16\/9 {
+    margin: 0 0 calc(var(--ld-sp-24) * -1 + 1px) !important;
+    padding-top: 56.25%;
+  }
+</style> -->
+
+## With scrollable content
+
+{% example %}
+<ld-bg-cells class="box5-16/9">
+  <ld-paragraph style="margin: var(--ld-sp-16)">
```

Comment:
Let's put the paragraph into a container with white background or not use text at all in the example. Otherwise we give a bad example in regards to accessibility. I know no one sane would implement such thing, but I've seen things you people wouldn't believe...

Reply:
Actually I'm not sure we even need this feature. I'd probably make the component as stupid as possible. In other words it should have as little responsibility and context assumptions as possible: It should just display cells in the background. And nothing else (such as acting as a container). I would leave such features to the developer who than simply implements a container as needed and puts the background cells component in its background. It would make the component more lightweight and simple to reason about so that you don't have to think about things such as "what about horizontal scrolling?" etc.